plugins/woocommerce/includes/wc-user-functions.php

```diff
@@ -462,8 +482,19 @@ function ( $status ) {
 ) AS subquery
 WHERE product_or_variation_id != 0
 ";
+			} else {
+				$sql = "
```

Comment:
```suggestion
				// HPOS: yes. Lookup table: no.
				$sql = "
```

Since it's not immediately obvious from the conditional clauses, I think it would be helpful to add a comment like this above each of the four SQL blocks stating which conditions it applies to.

Wdyt?

Reply:
Great idea! Thanks. Tackled in 24bd29f.